src/client/entity-editor/validators/common.ts

```diff
@@ -193,7 +194,7 @@ export function validateAuthorCreditRow(row: any): boolean {
 export const validateAuthorCreditSection = _.partialRight(
 	// Requires at least one Author Credit row
 	validateMultiple, _.partialRight.placeholder,
-	validateAuthorCreditRow, null, false
+	validateAuthorCreditRow, _.partialRight.placeholder, _.partialRight.placeholder
```

Comment:
Could you explain the double placeholder to me?
I don't understand what's happening

Reply:
This should help - 
second placeholder is for isOptional paramter of validateAuthorCreditRow to check if row can contain null author.
third placeholder is for requiredOneOrMore paramter of validatMultiple which should be used by server to validate optional AC.